doc/source/user/tutorial-svd.rst

```diff
@@ -266,6 +266,9 @@ have a second axis. Executing
 ::
 
     >>> s @ Vt
+    Traceback (most recent call last):
+      ...
+    ValueError: matmul: Input operand 1 has a mismatch in its core dimension 0, with gufunc signature (n?,k),(k,m?)->(n?,m?) (size 1024 is different from 768)
```

Comment:
line length > 80. [doctest is fine](https://docs.python.org/3/library/doctest.html#what-about-exceptions) with breaking the detail into multiple lines

Reply:
fixed.